challenge.js

```diff
@@ -6,7 +6,7 @@
  *		to a web page without refreshing it?
  *
  *	Answer below.
- */
+ */ I would do it through the console of the webpage by entering document.createElement.
```

Comment:
Yes, `document.createElement` will create a new HTML element object. However, it will not attach it to the DOM; it will be a floating, fully-functional HTML element that is only accessible via JavaScript. 1